src/wifi.rs

```diff
@@ -1050,6 +1073,41 @@ impl<'d> WifiDriver<'d> {
         esp!(unsafe { esp_wifi_set_mac(interface.into(), mac.as_ptr() as *mut _) })
     }
 
+    /// Enable and start WPS
+    pub fn start_wps(&mut self, config: &WpsConfig) -> Result<(), EspError> {
+        let config = Newtype::<esp_wps_config_t>::try_from(config)?;
+
+        self.set_configuration(&Configuration::None)?;
+        esp!(unsafe { esp_wifi_set_mode(wifi_mode_t_WIFI_MODE_STA) })?;
+
+        if !self.is_started()? {
+            self.start()?;
```

Comment:
Should we start the wifi implicitly? AFAIK we don't do that for other methods like `connect`.